go.mod

```diff
@@ -3,118 +3,120 @@ module github.com/GoogleCloudPlatform/prometheus-engine
 go 1.18
 
 require (
-	cloud.google.com/go/compute v1.5.0
-	cloud.google.com/go/monitoring v1.5.0
-	github.com/go-kit/log v0.2.0
+	cloud.google.com/go/compute/metadata v0.2.2
```

Comment:
Were all these changes auto-bumped as a result of the Prometheus upgrade followed by z `make presubmit`?